internal/incident/incident.go

```diff
@@ -278,15 +276,13 @@ func (i *Incident) processSeverityChangedEvent(ctx context.Context, tx *sqlx.Tx,
 		Message:     utils.ToDBString(ev.Message),
 	}
 
-	historyId, err := i.AddHistory(ctx, tx, history, true)
+	_, err := i.AddHistory(ctx, tx, history, true)
```

Comment:
Since we no longer need to insert and retrieve the history ID here, please change the last arg to `false`.